Makefile

```
@@ -1,3 +1,17 @@
+APPLICATION_NAME_1 = app
+APPLICATION_NAME_2 = telegram
+
+
 env:  ##@Environment Create .env file with variables
 	@$(eval SHELL:=/bin/bash)
-	@cp .env.sample .env
\ No newline at end of file
+	@cp .env.sample .env
+
+lint:  ##@Code Check code with pylint
+	poetry run python3 -m pylint $(APPLICATION_NAME_1)
+	poetry run python3 -m pylint $(APPLICATION_NAME_2)
+
+format:  ##@Code Reformat code with isort and black
+	poetry run python3 -m isort $(APPLICATION_NAME_1)
+	poetry run python3 -m black $(APPLICATION_NAME_1)
+	poetry run python3 -m isort $(APPLICATION_NAME_2)
```

```suggestion
CODE = app telegram

lint:
    pylint $(CODE)

format: 
    isort $(CODE)
    black $(CODE)
```

останется разобраться почему без poetry run python3 не запускается

---

и почему сразу с двумя аргументами в CODE оно не запускается